pkg/whip/whip_handler.go

```diff
@@ -57,7 +56,6 @@ type whipHandler struct {
 	rtcConfig          *rtcconfig.WebRTCConfig
 	pc                 *webrtc.PeerConnection
 	sync               *synchronizer.Synchronizer
```

Comment:
I believe the synchronizer struct member is now unused. It is passed in `newWHIPTrackHandler` which does nothing with it. The patch currently relies entirely on the gstreamer depacketizer to generate timestamps. Please see discussion on slack, but I believe we need "something" outside of gstreamer to generate timestamps that are consistent between audio and video here.